README.md

```diff
@@ -6,6 +6,12 @@ MKL.jl is a Julia package that allows users to use the Intel MKL library for Jul
 
 This package requires Julia 1.7+
 
+## Warning: `using MKL` can be breaking for uses of Base.LinearAlgebra
+
+MKL.jl is an incomplete BLAS wrapper which does not implement all of the functions of the [libblastrampoline](https://github.com/staticfloat/libblastrampoline) interface. Because of this,
```

Comment:
This is not really accurate. The issue, as I understand it, is that OpenBLAS and OpenBLAS uses different naming conventions for ILP64. All the functions are there.